tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -227,8 +225,9 @@ def test_host_sorting_based_on_recommendation_count():
 
 @pytest.mark.run_in_one_thread
 @pytest.mark.tier2
+@pytest.mark.rhel_ver_list([8])
```

Comment:
no rhel9? If this is supposed to work on all 8+ versions, then i'd recommend using the other, regex selector:
`@pytest.mark.rhel_ver_match('^[^67]')` # to select all single-digit versions but 6 or 7

Reply:
according to our external discussion - i'm not going to block this PR on this. IT might be expanded in the future